Gemfile.lock

```diff
@@ -53,6 +53,7 @@ GEM
     fastimage (1.8.1)
       addressable (~> 2.3, >= 2.3.5)
     ffi (1.9.10)
+    ffi (1.9.10-x64-mingw32)
```

Comment:
I've got Photoshop on my windows machine, so I think this is just cruft from installing windows gems